zipkin-web/src/main/scala/com/twitter/zipkin/web/Handlers.scala

```diff
@@ -383,4 +385,11 @@ class Handlers(mustacheGenerator: ZipkinMustache, queryExtractor: QueryExtractor
           .map(renderTrace(_))
       } getOrElse NotFound
     }
+
+  def handleServices(client: HttpClient, webServiceNameCacheTimeoutInSec: Int): Service[Request, Renderer] =
```

Comment:
lets call this servicesMaxAge - much shorter and the max-age concept in http caching is already seconds. sg?